### PR TITLE
[WIP] To stop using displayNames in components.

### PR DESCRIPTION
**Pull request by zacck**
Associated Issue: #3771

### Summary of Changes

* remove displayName from Preview Component

### Test Plan

Ran the dev server and checked that the application is running normally

### CODE REVIEW

File: src/components/Editor/Preview/index.js [hunk 1]
```diff
@@ -106,8 +106,6 @@ class Preview extends PureComponent {
   }
 }
 
-Preview.displayName = "Preview";
```
Comment (jasonLaster) [inline src/components/Editor/Preview/index.js]: can we try to remove all the displayName propeties in this PR?

Comment (zacck) [reply src/components/Editor/Preview/index.js]: yes going to do all of them in this one

**Comment by jasonLaster** [comment]
@zacck i think you'll need to fix yarn lint-js :)

https://circleci.com/gh/devtools-html/debugger.html/7024?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by zacck** [comment]
Will be downgrading my yarn tonight and will try again tomorrow

**Comment by zacck** [comment]
Ok so I fixed the eslint rule now back to removing display names

**Comment by zacck** [comment]
With this I think  I removed all the displayNames from the components. @jasonLaster I think this is done ...

**Comment by nyrosmith** [comment]
@zacck Can you resolve the conflict so I can review?

File: src/components/shared/Button/PaneToggle.js [hunk 2]
```diff
@@ -1,5 +1,3 @@
-// @flow
```
Comment (jasonLaster) [inline src/components/shared/Button/PaneToggle.js]: why remove this?

File: src/components/SecondaryPanes/Scopes.js [hunk 3]
```diff
@@ -82,7 +64,6 @@ class Scopes extends PureComponent {
             // TODO: See https://github.com/devtools-html/debugger.html/issues/3555.
             getObjectEntries={actor => {}}
             loadObjectEntries={grip => {}}
-            openLink={openLink}
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Scopes.js]: hmm, looks like the rebase got botched

File: src/components/SecondaryPanes/Scopes.js [hunk 4]
```diff
@@ -9,26 +9,13 @@ import {
...
-import type { Pause, LoadedObject, Frame } from "debugger-html";
-
-type Props = {
-  pauseInfo: Pause,
-  loadedObjects: LoadedObject[],
-  loadObjectProperties: Object => void,
-  selectedFrame: Frame,
-  frameScopes: ?any,
-  openLink: string => void
-};
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Scopes.js]: hmm... is it still botched?

Comment (zacck) [reply src/components/SecondaryPanes/Scopes.js]: no hold on

**Pull request merged**